modules/interaction/src/blaze/interaction/read.clj

```diff
@@ -50,6 +50,11 @@
 (defn- handler-intern [node]
   (fn [{{{:fhir.resource/keys [type]} :data} ::reitit/match
         {:keys [id vid]} :path-params}]
+    (log/debug
+      (if vid
+        (format "Get [base]/%s/%s/_history/%s" type id vid)
+        (format "GET [base]/%s/%s" type id))
+      )
```

Comment:
the parenthesis should not be not on a new line

Reply:
Corrected